hooks/post_gen_project.py

```diff
@@ -76,7 +76,7 @@ def main(initialise_git_repository: str) -> int:
                 "https://docs.github.com/en/get-started/quickstart/create-a-repo.\n\n"
                 "Then run:\n\n"
                 "git remote add origin git@github.com:"
-                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git"
+                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git",
```

Comment:
```suggestion
                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git"
```
I'm confused, we're suppressing [COM812](https://docs.astral.sh/ruff/rules/missing-trailing-comma/) but it looks like it's been applied here.

Reply:
Hmm, if linting is passing then might be a bug?

Reply:
Or I ran it first, before ignoring the rule. Will remove

Reply:
Our bike-shed is nice and waterproof now.